index.html

```diff
@@ -682,18 +682,24 @@ <h4>Registration Information</h4>
                 </section>
                 <section id="exploration-directory-anonymous-td" class="normative">
                     <h4>Anonymous TD Identifiers</h4>
-                    <a>Anonymous TD</a>s have no `id` (or `@id`) and are considered as blank nodes [[JSON-LD11]].
-                    The server assigns a local identifier to such TDs to allow management and retrieval
-                    from the directory.
+                    <a>Anonymous TD</a>s are considered as JSON-LD blank nodes [[JSON-LD11]].
+                    The server assigns a local identifier to such TDs
+                    to enable management and retrieval from the directory.
                     <span class="rfc2119-assertion" id="tdd-reg-anonymous-td-identifier">
                         In situations where the server exposes an Anonymous TD, it MUST
```

Comment:
MUST means always, I thought it was to be embedded only in enriched form?  I disagree with always embedding the blank node id.